src/var.jl

```diff
@@ -5,17 +5,27 @@ function polyvecvar(::Type{PV}, prefix, idxset) where {PV}
     [PV("$(prefix * string(i))") for i in idxset]
 end
 
+function polymatrixvar(::Type{PV}, prefix, rowidxset, colidxset) where {PV}
+    [PV("$(prefix * string(i) * "_" * string(j))") for i in rowidxset, j in colidxset]
```

Comment:
I choose an `_` to separate row and column indices, i.e., `x1_2`. Is this okay or do you have a better idea?

Reply:
I don't see any better idea